src/BinSkim.Rules/PERules/BA2024.EnableSpectreMitigations.cs

```diff
@@ -145,6 +145,9 @@ public override void AnalyzePortableExecutableAndPdb(BinaryAnalyzerContext conte
                 Symbol om = omView.Value;
                 ObjectModuleDetails omDetails = om.GetObjectModuleDetails();
 
+                // Debug libraries are not Spectre-mitigated by design.
```

Comment:
Small nit for fast readability and consistency:
I'd rather see:

if (.....)
{
    continue;
}

Reply:
fixed.